static/app/views/starfish/views/screens/screenLoadSpans/charts.tsx

```diff
@@ -149,6 +151,15 @@ export function ScreenCharts({yAxes, additionalFilters}: Props) {
     return <LoadingContainer />;
   }
 
+  if (!defined(primaryRelease) && !isReleasesLoading) {
+    Sentry.captureException(new Error('Screen summary missing releases'));
```

Comment:
this is a side effect, so it should be in a `useEffect`.

Reply:
https://github.com/getsentry/sentry/pull/60286/commits/9613185907dc65d2170a1698268b4af8977c095c